{"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError"]]},"schemaVersion":{"patch":0,"minor":3,"major":0},"metadata":{"symbolKind":"property","role":"symbol","title":"context","externalID":"s:8SotoCore14AWSClientErrorV7contextAA15AWSErrorContextVSgvp","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"context"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"modules":[{"name":"SotoCore"}],"roleHeading":"Instance Property"},"kind":"symbol","identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError\/context","interfaceLanguage":"swift"},"sections":[],"abstract":[{"text":"additional context information related to the error","type":"text"}],"variants":[{"paths":["\/documentation\/sotocore\/awsclienterror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"text":"?","kind":"text"}],"platforms":["macOS"]}]}],"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"AWSErrorContext","kind":"identifier"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"title":"AWSErrorContext","kind":"symbol","type":"topic","abstract":[{"text":"Additional information about error","type":"text"}],"url":"\/documentation\/sotocore\/awserrorcontext"},"doc://codes.soto.soto-core/documentation/SotoCore":{"abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}],"title":"SotoCore","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","url":"\/documentation\/sotocore","kind":"symbol","type":"topic","role":"collection"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClientError":{"kind":"symbol","abstract":[],"navigatorTitle":[{"kind":"identifier","text":"AWSClientError"}],"title":"AWSClientError","url":"\/documentation\/sotocore\/awsclienterror","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError","type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AWSClientError"}],"role":"symbol"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClientError/context":{"abstract":[{"type":"text","text":"additional context information related to the error"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError\/context","title":"context","type":"topic","url":"\/documentation\/sotocore\/awsclienterror\/context","fragments":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"kind":"symbol","role":"symbol"}}}